### ISSUE TITLE
No module named 'discord'

### ISSUE THREAD

**Issue by CrackedLegend** [Mattlau04/Dank-memer-coin-bomb-leecher#3]
I get the error:
ModuleNotFoundError: No module named 'discord'

Idk what to install

**Comment by mrpodato** [comment]
idk what to install too, pls help

**Comment by iiLeafy** [comment]
Bruh, open up command prompt and type this:
pip install discord.py
and done.

**Comment by CrackedLegend** [comment]
Of course... I did that! But didn't work anywaysThis can be closed now I'm not even using it anymore

**Comment by dirt3009** [comment]
do pip install os then do os.remove /

it will work, trust me bro 🇮🇳

**Comment by podato** [comment]
Bro i do that and it shutoof computa and i cannot turn on @podato @dirt3009 @

- help fix

- i want meme coins 
**help me zagerq
![fart](https://user-images.githubusercontent.com/61413317/93073406-3b798a80-f6c6-11ea-85a9-deb98fb0f3d3.png)
**
